src/classes/CW_BatchJobProgressPageController.cls

```diff
@@ -0,0 +1,92 @@
+/**
+* File Name   :    CW_BatchJobProgressPageController.cls
+* Description :    B-110984
+*                  show progress of CSV upload
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 09/04/2014 Klea Kolaric	  Initial version
+* 0.2 20/05/2015 Alex Panka     Adding Next page 
+*
+*/
+public with sharing class CW_BatchJobProgressPageController {
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.